README.md

```diff
@@ -13,15 +22,71 @@ notebooks in this repository:
 1. Install [Anaconda](https://docs.anaconda.com/anaconda/install/)
    or [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
 1. Navigate to your local copy of this repository.
-1. Run the script `env.sh` to create an Anaconda environment in the directory
-   `./env`:
+1. Run the script `env.sh` to create an Anaconda environment in the directory `./env`:
    ```console
    $ bash env.sh
    ```
    Note: This script takes a while to run.
 1. Activate the new environment and start JupyterLab:
    ```console
    $ conda activate ./env
-   $ jupyter lab
+   $ jupyter lab --debug
    ```
 
+#### Configuring a local Kubeflow Pipeline runtime
+
+[Elyra's Notebook pipeline visual editor](https://elyra.readthedocs.io/en/latest/getting_started/overview.html#notebook-pipelines-visual-editor)
+currently supports running these pipelines in a Kubeflow Pipeline runtime,
+and, if required, these are [the steps to install a local deployment of KFP](https://elyra.readthedocs.io/en/latest/developer_guide/deploying-kubeflow-locally-for-dev.html).
+
+After installing your Kubeflow Pipeline runtime, use the command below (with proper updates) to configure the new
+KFP runtime with Elyra.
+
+```bash
+elyra-metadata install runtimes --replace=true \
+       --schema_name=kfp \
+       --name=kfp-local \
+       --display_name="Kubeflow Pipeline (local)" \
+       --api_endpoint=http://cloning1.fyre.ibm.com:31380/pipeline \
+       --cos_endpoint=http://cloning1.fyre.ibm.com:31671 \
+       --cos_username=minio \
+       --cos_password=minio123 \
+       --cos_bucket=covid
+``` 
+
+**Note:** The cloud object storage above is a local minio object storage but others cloud based COS could be
+configured and used in this scenario.
+
+## Elyra Notebook pipelines
+
+Elyra provides a visual editor for building Notebook-based AI pipelines, simplifying the conversion of 
+multiple notebooks into batch jobs or workflows. By leveraging cloud-based resources to run their 
+experiments faster, the data scientists, machine learning engineers, and AI developers are then more productive,
+allowing them to spend their time using their technical skills.
+
+![Notebook pipeline](https://raw.githubusercontent.com/elyra-ai/community/master/resources/blog-announcement/elyra-pipelines.gif)
+
+### Running the Elyra pipeline
+
+The `us_data.pipeline`, which is located in the `pipeline` directory, can be run by click on the `play`
+button as seen on the image above. The `submit` dialog will request two inputs from the user, a name for
+the pipeline and a runtime to use while executing the pipeline. The list of available runtimes comes from
+the registered kfp runtimes documented above. After submission, Elyra will show a dialog with a direct
+link to where the experiment is being executed on kfp. 
+
+The user can access the pipelines and respective experiment runs via the `api_endpoint` of the KFP runtime
+(e.g. `http://cloning1.fyre.ibm.com:31380/pipeline`)
```

Comment:
Again, hard link to internal IBM address and port number. Users will need more explicit instructions to figure out how to access their own Kubeflow clusters. As stated above, I recommend that the primary thread of your instructions assume the user is using their laptop to run Kubeflow and provide examples based off that assumption.